docs/reference/metadata.md

```diff
@@ -980,7 +987,7 @@ Example:
   "references": [{
       "reference": "Nielsen et al,..",
       "identifier": "10.1234/foo.bar",
-      "scheme": "doi"
+      "scheme": "other"
```

Comment:
Should DOI be supported? In fact, it would be nice if someone who knows could provide the definition for this field because on our side we are unsure what it means if DOIs are not supported but funder ids are...